### PR TITLE
Add tutorial on craftable spawneggs

### PR DESCRIPTION
**Pull request by GitWither**

### CODE REVIEW

File: knowledge/craftable_spawneggs.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+layout: page
+title: craftable_spawners
+parent: Knowledge
+---
+ 
+# Using MoLang to create craftable spawn-eggs
+ 
+Minecraft Bedrock 1.16 introduced the ability to use MoLang expressions in the `data` field of recipe results. We can use the `query.get_actor_info_id` function to get the runtime integer ID of a given entity.
+
+
```
Comment (SirLich) [inline knowledge/craftable_spawneggs.md]: Can you include an example for custom entities as well? Or at least a note that its possible?

File: knowledge/craftable_spawneggs.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+layout: page
+title: craftable_spawners
+parent: Knowledge
```
Comment (SirLich) [inline knowledge/craftable_spawneggs.md]: I think this should go in the tutorials section. I will be re-organizing stuff soon, but for now I'm trying to keep stuff like this in tutorials. 

Knowledge is for "what is this thing"
Tutorials is for "how do I do this thing"

File: knowledge/craftable_spawneggs.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+---
+layout: page
+title: craftable_spawners
```
Comment (SirLich) [inline knowledge/craftable_spawneggs.md]: Title is used to create the side-bar. URL is built from the actual filename.

Lets change title to: Craftable Spawn-Eggs

**Review by SirLich — COMMENTED** [review]
Might be worth making a link to the "Egg Drops" doc, since thats very related. Anyway, thanks for the PR, and if you don't want to fix this stuff I will probably merge in and fix myself later down the line.

**Comment by GitWither** [comment]
All suggestions implemented 👍

**Pull request merged**